tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousCSTest.java

```diff
@@ -78,20 +79,9 @@ public static WebArchive deploy() {
      */
     @Test
     public void testAsyncIsNotFinished() {
-        CompletableFuture<Void> waitingFuture = newWaitingFuture();
-        
-        CompletionStage<Connection> resultFuture = client.serviceCS(waitingFuture);
-
-        Assert.assertFalse(completesQuickly(resultFuture));
-
-        complete(waitingFuture);
-
-        try {
-            waitUntilCompleted(resultFuture);
```

Comment:
It would be good to keep this cleanup logic (though it could possibly be put into a common `@AfterMethod` method) as otherwise you may be leaving a thread waiting forever.